react/admin/CostCenterDetails.tsx

```diff
@@ -445,18 +447,31 @@ const CostCenterDetails: FunctionComponent = () => {
       </PageBlock>
 
       {/* New Address Modal */}
-      <ModalDialog
+      <Modal
         centered
-        confirmation={{
-          onClick: () => handleAddNewAddress(),
-          label: formatMessage(messages.add),
-          disabled: !isValidAddress(newAddressState),
-        }}
-        cancelation={{
-          onClick: () => handleCloseModals(),
-          label: formatMessage(messages.cancel),
-        }}
-        loading={loadingState}
+        bottomBar={
+          <div className="nowrap">
+            <span className="mr4">
+              <Button
+                variation="tertiary"
+                onClick={() => handleCloseModals()}
+                isLoading={loadingState}
```

Comment:
disabled={loadingState}

Reply:
Good suggestion! I've committed the change and relinked the app.